modules/sdk/gradle-plugins/src/main/java/com/liferay/gradle/plugins/LiferayOSGiExtPlugin.java

```diff
@@ -222,11 +223,34 @@ private void _configureProject(
 		}
 	}
 
-	private void _configureTaskDeploy(Jar jar) {
+	private void _configureTaskDeploy(Project project, Jar jar) {
 		Copy copy = (Copy)GradleUtil.getTask(
 			jar.getProject(), LiferayBasePlugin.DEPLOY_TASK_NAME);
 
-		copy.from(jar);
+		final LiferayExtension liferayExtension = GradleUtil.getExtension(
```

Comment:
This variable is only used in the nested closure, so please move it there to follow the `as used` principle